lib/chef/resource/chocolatey_source.rb

```diff
@@ -23,10 +23,17 @@ class ChocolateySource < Chef::Resource
       description "Use the chocolatey_source resource to add or remove Chocolatey sources."
       introduced "14.3"
 
-      property :source_name, String, name_property: true
-      property :source, String
-      property :bypass_proxy, [TrueClass, FalseClass], default: false
-      property :priority, Integer, default: 0
+      property :source_name, String, name_property: true,
+               description: "The name of the source to add. We'll use the resource's name if this isn't provided."
+
+      property :source, String,
+               description: "The source URL."
+
+      property :bypass_proxy, [TrueClass, FalseClass], default: false,
+               description: "Whether or not to bypass the system's proxy settings to access the source."
```

Comment:
Language looks inconsistent with other Truthy/Falsy descriptions.  Other descriptions start `should` and are formed as a question.  We should pick one style over the other.